src/rotator_library/providers/antigravity_validators.py

```
@@ -0,0 +1,224 @@
+    Args:
+        reasoning_effort: Optional reasoning effort level
+        
+    Raises:
+        ValueError: If reasoning_effort is invalid
+    """
+    if reasoning_effort is not None:
+        valid_levels = {'low', 'medium', 'high'}
+        if reasoning_effort not in valid_levels:
+            raise ValueError(
+                f"reasoning_effort must be one of {', '.join(valid_levels)}, got '{reasoning_effort}'"
+            )
```

Ruff (0.14.6)

Avoid specifying long messages outside the exception class

(TRY003)